website/docs/docs/use-dbt-semantic-layer/dbt-sl.md

```diff
@@ -0,0 +1,169 @@
+    link="/docs/use-dbt-semantic-layer/sl-architecture"
+    icon="dbt-bit"/>
+
+<Card
+    title="Semantic Layer API"
+    body="Use the Semantic Layer API to query metrics in downstream tools for consistent, reliable data metrics."
+    link="/docs/dbt-cloud-apis/sl-api-overview"
+    icon="dbt-bit"/>
+
+</div>
+
+</VersionBlock>
+
+<VersionBlock lastVersion="1.5">
+
+import LegacyInfo from '/snippets/_legacy-sl-callout.md';
+
+<LegacyInfo />
+
+The dbt Semantic Layer allows data teams to centrally define essential business metrics like `revenue`, `customer`, and `churn` in the modeling layer (your dbt project) for consistent self-service within downstream data tools like BI and metadata management solutions. The dbt Semantic Layer provides the flexibility to define metrics on top of your existing models and then query those metrics and models in your analysis tools of choice.
+
+The result? You have less duplicate coding for data teams and more consistency for data consumers. 
+
+The dbt Semantic Layer has four main parts:
+
+- Define your metrics in version-controlled dbt project code using [MetricFlow](/docs/build/about-metricflow) 
+	* dbt_metrics is now deprecated
+- Import your metric definitions via the [Discovery API](/docs/dbt-cloud-apis/discovery-api)
+- Query your metric data via the dbt Proxy Server
+- Explore and analyze dbt metrics in downstream tools
+
+### What makes the dbt Semantic Layer different?
+
+The dbt Semantic Layer reduces code duplication and inconsistency regarding your business metrics. By moving metric definitions out of the BI layer and into the modeling layer, data teams can feel confident that different business units are working from the same metric definitions, regardless of their tool of choice. If a metric definition changes in dbt, it’s refreshed everywhere it’s invoked and creates consistency across all applications. You can also use the dbt Semantic Layer to query models and use macros.
+
+
+## Prerequisites
+To use the dbt Semantic Layer, you’ll need to meet the following:
+
+<Snippet path="sl-prerequisites" />
+
+<Snippet path="sl-considerations-banner" />
+
+
+## Manage metrics
+
+:::info 📌
+
+New to dbt or metrics? Check out our [quickstart guide](/quickstarts) to build your first dbt project! If you'd like to define your first metrics, try our [Jaffle Shop](https://github.com/dbt-labs/jaffle_shop_metrics) example project.
+
+:::
+
+If you're not sure whether to define a metric in dbt or not, ask yourself the following: 
+
+> *Is this something our teams consistently need to report on?*  
+
+An important business metric should be:
+
+- Well-defined (the definition is agreed upon throughout the entire organization)
+- Time-bound (able to be compared across time)
+
+A great example of this is **revenue** &mdash; it can be aggregated on multiple levels (weekly, monthly, etc) and is key for the broader business to understand.
```

Comment:
i was taught that em dashes are used to set off phrases and clauses. since this is a complete sentence (not a phrase or clause), suggest using a period or a semicolon  

```suggestion
A great example of this is **revenue**. It can be aggregated on multiple levels (weekly, monthly, and so on) and is key for the broader business to understand.
```